core/src/test/scala/unit/kafka/coordinator/transaction/ProducerIdManagerTest.scala

```diff
@@ -38,22 +38,32 @@ import org.mockito.Mockito.{mock, when}
 import java.util.concurrent.{CountDownLatch, Executors, TimeUnit}
 import java.util.concurrent.atomic.AtomicBoolean
 import scala.collection.mutable
-import scala.util.{Failure, Success}
+import scala.util.{Failure, Success, Try}
 
 class ProducerIdManagerTest {
 
   var brokerToController: NodeToControllerChannelManager = mock(classOf[NodeToControllerChannelManager])
   val zkClient: KafkaZkClient = mock(classOf[KafkaZkClient])
 
+  class ErrorQueue(initialErrorCounts: Errors*) {
+    private val queue: mutable.Queue[Errors] = mutable.Queue.empty ++ initialErrorCounts
```

Comment:
It think this can just be

```suggestion
    private val queue: mutable.Queue[Errors] = mutable.Queue(initialErrorCounts: _*)
```